### PR TITLE
Setup PowerShell scripts to simplify development

### LINKED ISSUE #24: Setup PowerShell scripts to simplify development
As a developer, I want it to be as easy as possible to start developing right away.

### PR DESCRIPTION
**Pull request by b3h3m0th**
closes #24

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "nrwl.angular-console",
     "esbenp.prettier-vscode",
     "firsttris.vscode-jest-runner",
-    "dbaeumer.vscode-eslint"
+    "dbaeumer.vscode-eslint",
+    "angular.ng-template"
```
Comment (patrickcerny) [inline .vscode/extensions.json:7]: angular.ng-template in a react project?

**Review by patrickcerny — CHANGES_REQUESTED** [review]
Looks nice! Does this close the Docker issue? And please remove / explain the angular thingy there! All in all nice work!

**Comment by b3h3m0th** [comment]
Boa wieso schriban mir uf englisch?

**Comment by b3h3m0th** [comment]
Und na i krig des mit docker iwi ned gschissa. joa des docker issue kama eig closen.

**Comment by Georg-code** [comment]
We are international af

File: .vscode/extensions.json (hunk 1 above)
Comment (b3h3m0th) [reply .vscode/extensions.json:7]: lol isch recht

**Review by patrickcerny — APPROVED** [review]

**Pull request merged**